src/main/java/core/TestBaseHtmlUnit.java

```diff
@@ -0,0 +1,28 @@
+package core;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.htmlunit.HtmlUnitDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+import java.util.concurrent.TimeUnit;
+
+public class TestBaseHtmlUnit {
+    public static final String URL = "https://192.168.100.26";
+    private static final int DEFAULT_TIMEOUT = 10;
+    public static WebDriver driverForHTMLUnit;
```

Comment:
If you need WebDriver in another classes, it's better to create singleton for WebDriver